### ISSUE TITLE
what is the final loss value look like?

### ISSUE THREAD

**Issue by xingjy10** [NVIDIA/tacotron2#100]
i trained 82,500 and the Validation loss is 0.53.

**Comment by deepconsc** [comment]
We hit 0.17 validation loss by starting from pre-trained model, and after loss start drifting, we annealed the learning rate to 1e-5. It took total of 54K steps.
